models/demos/yolov4/tests/test_perf_yolo.py

```diff
@@ -23,7 +23,7 @@
 
 
 def get_expected_times():
-    return (40, 16.2)
+    return (400, 16.2)
```

Comment:
why 400?

Reply:
@shwetankTT The CI tests were failing due to compile time taking longer than expected. I tried to increase it and re-run the CI test . How do you determine the proper threshold value per model? It's been consistently failing with 40 on the CI tests: https://github.com/tenstorrent/tt-metal/actions/runs/12263940234/job/34271378440#step:10:1289

Reply:
Running locally and see what is the time I am getting + some extra time so that CI passes.

Reply:
hmm, it is still taking 40 in the[ latest CI test](https://github.com/tenstorrent/tt-metal/actions/runs/12263940234/job/34297873345#step:10:1290)! 

@shwetankTT any idea why?

Reply:
Not sure. 
Check the set up job line no 27 its taking an old commit of yours.

Reply:
The current run --> https://github.com/tenstorrent/tt-metal/actions/runs/12292953950/job/34304818967